### PR TITLE
Bug/update disabling models

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Pull Request
**Are you a current Fivetran customer?** 
Fivetran created PR

**What change(s) does this PR introduce?** 
## Bug Fixes
  - Add all model variables to the README "Disabling Models" section
  - Remove model variables from this package's `dbt_project.yml` to avoid potential conflict with a user's settings.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [link bug/feature number here] #34 
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🐧 
**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ vars:
...
 
 ### Disabling Models
-Your connector may not be syncing all tabes that this package references. This might be because you are excluding those tables. If you are not using those tables, you can disable the corresponding functionality in the package by specifying the variable in your `dbt_project.yml`. The metrics from the disabled tables will not populate in downstream models. By default, all packages are assumed to be true. You only have to add variables for tables you want to disable, like so:
+It is possible that your Salesforce connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Salesforce or actively excluded some tables from your syncs. 
 
-The `salesforce__user_role_enabled` variable below refers to the `user_role` table. 
+To disable the corresponding functionality in this package, you must add the corresponding variable(s) to your `dbt_project.yml`, which are listed below. By default, that is if none of these variables are added, all variables are assumed to be true. Add variables only for the tables you would like to disable:
 
 ```yml
 # dbt_project.yml
```
Comment (fivetran-avinash) [inline README.md:82]: You don't have to include lines 82-86 with the commented out dbt_project.yml and config-version: 2 line.

**Review by fivetran-avinash — COMMENTED** [review]
Hello @fivetran-catfritz! Everything looks good, just some minor tweaks to clean up the Readme.

One other thing--if you can update your integration_tests/requirements.txt files to the following specs below and include it in the PR. Joe mentioned that we should start updating requirements files going forward since the original 1.0.0 versions are starting to throw errors, just something to keep in mind!

Screen Shot 2022-09-26 at 2 31 50 PM

**Review by fivetran-avinash — COMMENTED** [review]
Hello @fivetran-catfritz! Everything looks good, just some minor tweaks to clean up the Readme.

One other thing--if you can update your integration_tests/requirements.txt files to the following specs below and include it in the PR. Joe mentioned that we should start updating requirements files going forward since the original 1.0.0 versions are starting to throw errors, just something to keep in mind!

Screen Shot 2022-09-26 at 2 31 50 PM

File: README.md (hunk 1 above)
Comment (fivetran-catfritz) [reply README.md]: Thanks, and I have updated. I also updated similarly in a few other similar code blocks for consistency.

**Comment by fivetran-catfritz** [comment]
@fivetran-avinash I have updated the pushed the requirements and readme changes!

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-catfritz Looks great!

**Pull request merged**